examples/geoscience-aus-cat/projects_geoscience-aus-cat_assets_ga_ls_wo_fq_cyear_3.js

```diff
@@ -1,4 +1,4 @@
-var water_obs = ee.ImageCollection('projects/geoscience-aus-cat/assets/annual-water-obs');
+var water_obs = ee.ImageCollection('projects/geoscience-aus-cat/assets/ga_ls_wo_fq_cyear_3');
```

Comment:
could you add a gray background layer to this example like it's done in preview? Otherwise the Google maps base layer shows through.